demo/components/wizard/index.html

```diff
@@ -1,15 +1,20 @@
-﻿<!doctype html>
+<!doctype html>
 <html lang="en">
 
 <head>
 	<meta charset="utf-8">
 	<meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
-	<link rel="stylesheet" href="/node_modules/@brightspace-ui/core/components/demo/styles.css" type="text/css">
+	<link rel="stylesheet" href="../../../node_modules/@brightspace-ui/core/components/demo/styles.css" type="text/css">
 	<script type="module">
-		import '@brightspace-ui/core/components/demo/demo-page.js';
+		import '../../../node_modules/@brightspace-ui/core/components/demo/demo-page.js';
 		import '../../../src/components/wizard/wizard.js';
 		import '../../../src/components/wizard/step.js';
 	</script>
+	<style>
+		d2l-labs-wizard {
+			line-height: 1.2rem;
+		}
+	</style>
```

Comment:
From what I can tell, the component doesn't actually work with our default line-height. I'm not going to touch the component right now, so fixing the demo.